### DISCUSSION TITLE
Questions about the different variants

### DISCUSSION THREAD

**Discussion by clemyan** [microsoft/cascadia-code#758] [Q&A] [answered]
1. Regarding the different font formats: I should install either TTF variable or TTF static but never both simultaneously, correct?
2. Regarding Cascadia Code vs Cascadia Mono: As Mono is described as Code without ligatures, if I disable ligatures from the app (e.g VSCode) using Cascadia, using Code vs Mono should be exactly the same, correct?

**Comment by aaronbell** [comment, marked as answer]
Hi sorry I think I missed this question.

1) Yes. Only install one or the other. Installing both will confuse your computer.

2) Yes, Cascadia Mono is identical to Cascadia Code when ligatures are disabled.
